frontend/src/components/DataCard/CardData.jsx

```diff
@@ -29,7 +30,7 @@ function drawSkipButton(card, onSkip) {
 
     const handleSkip = (event) => {
         event.preventDefault();
-        onSkip(card, message);
```

Comment:
Actually though, right now, the skip button exists in the history table, but what exactly does that mean? I don't think it should really render there.